src/nmcli/connections.c

```diff
@@ -3829,8 +3829,11 @@ check_valid_name_toplevel(const char *val, const char **port_type, GError **erro
         if (setting_info->alias)
             g_ptr_array_add(tmp_arr, (gpointer) setting_info->alias);
     }
+    g_ptr_array_add(tmp_arr, "bond-port");
     g_ptr_array_add(tmp_arr, "bond-slave");
+    g_ptr_array_add(tmp_arr, "bridge-port");
     g_ptr_array_add(tmp_arr, "bridge-slave");
+    g_ptr_array_add(tmp_arr, "team-port");
```

Comment:
Those `*-slave` connection types are now obsolete and should never be used. They implicitly create an "ethernet" connection and this is confusing because the name suggests it's just a port/slave that can be of any type. For this reason, it's always better to use the explicit `type * port-type *` syntax. All the documentation now should refer to the new, better way and I think we need to find a way to deprecate and remove those `*-slave` connection types.

Reply:
Dropped that change.